src/iface/fragments.rs

```diff
@@ -0,0 +1,170 @@
+use storage::Assembler;
+use time::Instant;
+use managed::ManagedSlice;
+
+/// Default timeout duration
+const FRAGMENTATION_TIMEOUT_MS: i64 = 500;
+
+#[allow(dead_code)]
+pub struct Packet<'a, T: 'a> {
+    assembler: Assembler,
+    pub rx_buffer: ManagedSlice<'a, T>,
```

Comment:
Why is this `pub`?

Reply:
because `process_ipv4_fragment` returns `return Ok(Some(Ipv4Packet::new_checked(&fragment.rx_buffer[0..front])?));`

So having a getter function that returns a slice from a given range seemed unnecessary. But I am happy to add it.

Reply:
Fixed.